process-engine/bpmn-studio
# 🐳 Add Docker Build Step

## Changes

1. Add Docker Build Step
2. Fix Webversion of BPMN Studio

PR: #1711

## How to test the changes

- Run:
```
docker pull 5minds/bpmn-studio-bundle:5.7.0-feature-50b58a-k03ppf0y
docker run -p 8000:8000 -p 9000:9000 5minds/bpmn-studio-bundle:5.7.0-feature-50b58a-k03ppf0y
```
- Open a browser
- Go to `localhost:9000`
- **Notice that the BPMN-Studio works and is connected to a  ProcessEngine**
